samples/drivers/adc/src/main.c

```diff
@@ -59,6 +59,7 @@ struct adc_sequence sequence = {
 	/* buffer size in bytes, not number of samples */
 	.buffer_size = sizeof(sample_buffer),
 	.resolution  = ADC_RESOLUTION,
+	.calibrate = false,
```

Comment:
This field was anyway initialized to 0, so the only difference is that now this is done explicitly in the source code. I'm not sure there is much value in this change, but for sure it won't fix #38578.